websites/F/Free Web Novel/dist/metadata.json

```
@@ -0,0 +1,94 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "BigBoi",
+		"id": "853932304257908746"
+	},
+	"service": "Free Web Novel",
+	"description": {
+		"en": "We are offering free books online read! Read novel updated daily: light novel translations, web novel, chinese novel, english Novel, japanese novel and korean novel online."
+	},
+	"url": "freewebnovel.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/ePGzX1S.png",
+	"thumbnail": "https://i.imgur.com/PJw9FKX.png",
+	"color": "#5a5a5a",
+	"category": "socials",
+	"tags": [
+		"novel",
+		"book",
+		"books",
+		"reading",
+		"free",
+		"nsfw"
+	],
+	"settings": [
+		{
+			"id": "showLogo",
```

But to be honest, why would anyone only want to display the logo?
```suggestion
			"id": "logoOnly",
```